Userland/Libraries/LibC/malloc.cpp

```diff
@@ -129,7 +129,7 @@ static Allocator* allocator_for_size(size_t size, size_t& good_size)
 #ifdef RECYCLE_BIG_ALLOCATIONS
 static BigAllocator* big_allocator_for_size(size_t size)
 {
-    if (size == 65536)
+    if (size == ChunkedBlock::block_size)
```

Comment:
The correlation between `ChunkedBlock::block_size` and the block size of the big allocator seems a bit flimsy.

Reply:
Alright, that's gone. I just thought those were the same intentionally and that they'd actually be related.